frontend/src/hooks/useUpdater.ts

```
@@ -0,0 +1,126 @@
+      return;
+    }
+
+    setIsDownloading(true);
+    setDownloadProgress({ downloaded: 0, total: 0 });
+    setError(null);
+
+    try {
+      await updateAvailable.downloadAndInstall((event) => {
+        switch (event.event) {
+          case 'Started':
+            const contentLength = event.data.contentLength || 0;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)